README.md

```diff
@@ -338,16 +385,20 @@ posts for further information
     </td><td><code>2</code></td></tr>
 
    <tr><td>appGatewayCertBlob</td><td>string</td>
-    <td>A Base-64 encoded form of the PFX certificate for the Application Gateway. 
+    <td>A Base-64 encoded form of the PKCS#12 archive (.p12/.pfx) certificate for the Application Gateway.
     This certificate is used to secure HTTPS connections to and from the Application Gateway.
     <strong>Required when selecting <code>gateway</code> for load balancing.</strong>
     </td><td><code>""</code></td></tr>
 
    <tr><td>appGatewayCertPassword</td><td>securestring</td>
-    <td>The password for the PFX certificate for the Application Gateway. Defaults to <code>""</code>.
+    <td>The password for the PKCS#12 archive (.p12/.pfx) certificate for the Application Gateway. Defaults to <code>""</code>.
     <strong>Required when selecting <code>gateway</code> for load balancing.</strong>
     </td><td><code>""</code></td></tr>
 
+  <tr><td>appGatewayEsHttpCertPublicKey</td><td>securestring</td>
+    <td>The Base-64 encoded public key (.cer) for the certificate used to secure the HTTP layer of Elasticsearch. Used by the Application Gateway to whitelist certificates used by the backend pool. Required when using <code>esHttpCertBlob</code> to secure the HTTP layer of Elasticsearch and selecting <code>gateway</code> for load balancing. <strong>X-Pack plugin must be installed</strong>
```

Comment:
`The Base-64 encoded public key (.cer) for the certificate ` == The certificate

Reply:
Will update 👍